_posts/2021-09-20-newpipe-0.21.10-released.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "NewPipe 0.21.10 released: Here's your order! Enjoy the Full Screen!"
+short: "NewPipe 0.21.10 released"
+date: 2021-09-10 17:00:00 + 1:00
+categories: [pinned, release]
+author:  <a href="https://github.com/opusforlife2">@Opusforlife</a> & <a href="https://github.com/poolitzer">@Poolitzer</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+This release brings you working imports from your YouTube subscriptions export, an "always go into full screen" toggle when you play a video via the main player, a new Invidious instance, and so much more that you basically have to read on.
+
+<!-- more -->
+
+## New
+
+- Rejoice! [@talanc](https://github.com/talanc) fixed an issue plaguing users since late July: Google changed the way it exports subscriptions from YouTube, rendering the previous import into NewPipe non-functional. You can now import your downloaded [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) file, or even the ZIP file that contains that CSV, directly into NewPipe!
+
+- The behaviour of long press on the player queue was streamlined by [@ktprograms](https://github.com/ktprograms): It now also displays the popup menu from the main player.
+
+- A cool new setting was added by [@Stypox](https://github.com/Stypox): You can now automatically have videos playing full screen if you use the main player.
+
+- [@Stypox](https://github.com/Stypox) also split search suggestions into two categories: local and remote. Now you have the option to enable or disable them independently of each other.
+
+- [@magicbrothers](https://github.com/magicbrothers) added a new Invidious instance, Tubus, meant for educational videos for children.
+
+- [@ktprograms](https://github.com/ktprograms) finished the ongoing work of adding the `Show channel details` option where it was still missing, that is, in local playlists, feed items, and history items.
```

Comment:
Probably better if double quotes were used instead, as this would be shown as code (i.e. `Show channel details`):
```suggestion
- [@ktprograms](https://github.com/ktprograms) finished the ongoing work of adding the "Show channel details" option where it was still missing, that is, in local playlists, feed items, and history items.
```